app/views/documents/fields/_govspeak_input.html.erb

```diff
@@ -1,5 +1,5 @@
 <div class="govuk-grid-row">
-  <div class="govuk-grid-column-two-thirds">
+  <div class="govuk-grid-column-two-thirds" id="edit-document-body">
```

Comment:
Oops something I forgot last review. This field is dynamic, it gets it's name from `field.id`

Reply:
@kevindew The field does, but I'm ID'ing to the containing frame rather than the field itself, to avoid these issues. Shouldn't need changing here, just shortening.

Reply:
I mean that this partial is intended to be used by fields other than "body"